### DISCUSSION TITLE
希望大佬加上“十周年谜题-或此或彼”的提示功能

### DISCUSSION THREAD

**Discussion by ioi301** [Pik-4/HsMod#206] [Ideas (许愿池)]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

“十周年谜题-或此或彼”，50个选择实在是太煎熬了。希望增加提示功能。

### Describe the solution you'd like/描述你想要的解决方案

类似这个视频下的提示(说的也是hsmod看起来也像，但最新版并未发现有此功能，不知是否是这个hsmod还是其他同名程序)
https://www.bilibili.com/video/BV1aj421D7MU/?spm_id_from=333.1391.0.0&vd_source=92b2fc2b07fe8ba24f3ff352424c5967

**Comment by Pik-4** [comment]
友商阿波罗的mod
